package.json

```diff
@@ -73,7 +73,7 @@
     "lunr": "2.3.6",
     "markdown-toc": "1.2.0",
     "node-sass": "4.13.1",
-    "prettier": "1.18.2",
+    "prettier": "2.8.4",
```

Comment:
(putting my comment here for the sake of having a threaded discussion)

Since you're updating prettier, could you also wrap the prettier script glob patterns in quotes?

```
"prettier": "prettier 'lib/**/*.ts' 'examples/**/*' --write",
```

I noticed they are wrongly extrapolated when passed through yarn. This will allow you to fix a few additional files as well.